static/app/stores/serverSideSamplingStore.tsx

```diff
@@ -0,0 +1,57 @@
+import {createStore} from 'reflux';
```

Comment:
Is this new file here on purpose or is it just a leftover from a different feature?

Reply:
I pushed it when solving conflict. good catch